python/examples/ola_artnet_params.py

```diff
@@ -26,9 +26,9 @@
 def ArtNetConfigureReply(state, response):
   reply = ArtNetConfigMessages_pb2.Reply()
   reply.ParseFromString(response)
-  print 'Short Name: %s' % reply.options.short_name
-  print 'Long Name: %s' % reply.options.long_name
-  print 'Subnet: %d' % reply.options.subnet
+  print(('Short Name: %s' % reply.options.short_name))
```

Comment:
Try this:

sed -i '' "s/print((\(.*\)))/print(\1)/" *.py

You may have to fix the multi-line ones manually.